templates/base.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {% block title %}
+        <title>Taxi service</title>
+    {% endblock %}
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css" integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
+
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+</head>
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-sm-3">
+              {% block sidebar %}
+                  {% include "includes/sidebar.html" %}
+              {% endblock %}
+            </div>
+            <div class="col-sm-9">
+                <h1>Taxi Service</h1>
+                <p>Welcome to the best taxi service</p>
+                <h3>Taxi service content</h3>
+                {% block content %}
+                    {% include "taxi/index.html" %}
+                {% endblock %}
+            </div>
+        </div>
+    </div>
+</body>
+</html>
```

Comment:
same is here, fix this everywhere)

Reply:
![image](https://github.com/mate-academy/py-taxi-service-home-page/assets/84961728/87d051cf-53d2-4484-8461-2be2b1bd132f)
![image](https://github.com/mate-academy/py-taxi-service-home-page/assets/84961728/5cd0403a-c52f-4012-8573-808015fa6f13)
![image](https://github.com/mate-academy/py-taxi-service-home-page/assets/84961728/a12f6028-7164-434a-b81d-b7fd04343435)